sdk/servicebus/azure-servicebus/tests/async_tests/mgmt_tests/test_mgmt_queues_async.py

```diff
@@ -260,6 +260,13 @@ async def test_async_mgmt_queue_create_with_queue_description(self, servicebus_n
             max_size_in_megabytes=3072,
             requires_session=True
         )
+
+        with pytest.raises(HttpResponseError):
```

Comment:
is this also testing that `max_message_size_in_kilobytes` is > 1024? if so, doesn't the test `test_async_mgmt_queue_premium_create_with_queue_description` below cover this scenario?

Reply:
this test case is to test that `max_message_size_in_kilobytes` isn't supported on standard tier

Reply:
test updated to 1024